templates/route53/7/rancher-compose.yml

```diff
@@ -0,0 +1,47 @@
+.catalog:
+  name: "Route53 DNS"
+  version: "v0.5.0-rancher1"
+  description: "Rancher External DNS service powered by Amazon Route53. Requires Rancher version 0.44.0"
+  minimum_rancher_version: v1.1.0
+  questions:
+    - variable: "AWS_ACCESS_KEY"
+      label: "AWS Access Key ID"
+      description: "Access key ID for your AWS account"
+      type: "string"
+      required: true
+    - variable: "AWS_SECRET_KEY"
+      label: "AWS Secret Access Key"
+      description: "Secret access key for your AWS account"
+      type: "string"
+      required: true
+    - variable: "AWS_REGION"
+      label: "AWS Region"
+      description: "AWS region name"
+      type: "string"
+      default: "us-west-2"
+      required: true
+    - variable: "TTL"
+      label: "TTL"
+      description: "The resource record cache time to live (TTL), in seconds"
+      type: "int"
+      default: 60
+      required: false
+    - variable: "ROOT_DOMAIN"
+      label: "Hosted Zone Name"
+      description: "Route53 hosted zone name (zone has to be pre-created). DNS entries will be created for <service>.<stack>.<environment>.<hosted zone>"
+      type: "string"
+      required: true
+    - variable: "ROUTE53_ZONE_ID"
```

Comment:
A little late now that it's baked in to the image, but this naming is odd.. Other fields are not prefixed ROUTE53, and the hosted zone is called ROOT_DOMAIN above.  `ROOT_DOMAIN_ID`?